contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2767,7 +2791,11 @@ rewriteBatchLevelStatement(
 		{
 			auto options = cctx->trigger_option();
 			auto commas = cctx->COMMA();
-			GetTokenFunc<TSqlParser::Trigger_optionContext*> getToken = [](TSqlParser::Trigger_optionContext* o) { return o->SCHEMABINDING(); };
+			GetTokenFunc<TSqlParser::Trigger_optionContext*> getToken = [](TSqlParser::Trigger_optionContext* o) {
```

Comment:
Do we have test cases for both dml and ddl triggers?

Reply:
Test case for dml triggers were included, but for ddl isn't included as DDL is not currently supported in BBF